modules/displayTime.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] Please avoid making custom changes in disabling `eslint`.
If you have one function to export, then at the end of document you can write 

`export default myFunction`
then import it:
`import myFunction from './modules/index.js'`